data/input/court.usability.js

```diff
@@ -29,11 +29,11 @@ module.exports = {
   },
```

Comment:
Now I'm wondering... maybe we just completely remove `usability`—opening up an open Rinkeby test environment is something that we can re-evaluate after launching Agreements.

Could we just maintain `staging` as the "fast test" environment?

Reply:
Yea, I think having two of these is more than enough, one mimicking mainnet and the other one faster for the daily basis